ethereum_history_api/circuits/lib/src/receipt.nr

```diff
@@ -1,6 +1,6 @@
 use crate::misc::{types::{Bytes32, Address}, option::make_option};
 use crate::header::get_header;
-use crate::verifiers::receipt::verify_receipt;
+use crate::verifiers::receipt::{MAX_ENCODED_RECEIPT_LENGTH, verify_receipt};
```

Comment:
Will replace import of `MAX_ENCODED_RECEIPT_LENGTH` with generic parameter `MAX_RECEIPT_ENCODED_LEN` in a separate PR